lib/test_suite/analyzer_verification/assert_call.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ElixirAnalyzer.TestSuite.AnalyzerVerification.AssertCall do
+  @dialyzer generated: true
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Pacman Rules
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_call "finds call to local helper function" do
+    type :informational
+    called_fn :local, &AssertCallVerification.helper/0
+    comment "didn't find a local call to helper/0"
+  end
+
+  assert_call "finds call to private local helper function" do
+    type :informational
+    called_fn :local, &AssertCallVerification.private_helper/0
+    comment "didn't find a local call to private_helper/0"
+  end
+
+  assert_call "finds call to local helper function within function" do
+    type :informational
+    calling_fn :global, &AssertCallVerification.function/0
+    called_fn :local, &AssertCallVerification.helper/0
+    comment "didn't find a local call to helper/0 within function/0"
+  end
+
+  assert_call "finds call to private local helper function within function" do
+    type :informational
+    calling_fn :global, &AssertCallVerification.function/0
+    called_fn :local, &AssertCallVerification.private_helper/0
+    comment "didn't find a local call to private_helper/0 within function"
+  end
+end
```

Comment:
The purpose of this module is to serve as a basis for the test `ElixirAnalyzer.ExerciseTest.AssertCallTest`, right? It's not actually part of the source code of the analyzer. If that's the case, I think it belongs in `test/support` instead.